views/static/registration.php

```diff
@@ -1,3 +1,37 @@
+<?php
+require_once('connection.php');
+$link = Db::getInstance();
+
+// If the values are posted, insert them into the database.
+if (isset($_POST['emailAddress']) && isset($_POST['password_1'])) {
+    $firstName = $_POST['firstName'];
+    $lastName = $_POST['lastName'];
+    $emailAddress = $_POST['emailAddress'];
+    $phoneNumber = $_POST['phoneNumber'];
+    $numOfKids = $_POST['numOfKids'];
+    $userRole = $_POST['userRole'];
+    $password = $_POST['password_1'];
+
+    $query = "SELECT * FROM Users WHERE email='$emailAddress'";
+    $result = $link->query($query);
+    $count = mysqli_num_rows($result);
+//3.1.2 If the posted values are equal to the database values, then session will be created for the user.
+    if ($count == 0) {
+        $query = "INSERT INTO Users(email, password, firstName, lastName, phoneNumber, notificationState, UserTypeId) VALUES ('$emailAddress', '$password', '$firstName', '$lastName', '$phoneNumber', 1, 1)";
```

Comment:
are usertypeid and notificationstate meant to be hard coded?

Reply:
This is for testing purpose. I will work on this as well as password encryption and user profile.